django_orghierarchy/models.py

```diff
@@ -80,17 +81,20 @@ class Organization(MPTTModel, DataModel):
     name = models.CharField(max_length=255, help_text=_('A primary name, e.g. a legally recognized name'))
     founding_date = models.DateField(blank=True, null=True, help_text=_('A date of founding'))
     dissolution_date = models.DateField(blank=True, null=True, help_text=_('A date of dissolution'))
-    parent = TreeForeignKey('self', null=True, blank=True, related_name='children',
+    parent = TreeForeignKey('self', on_delete=models.CASCADE, null=True, blank=True, related_name='children',
                             help_text=_('The organizations that contain this organization'))
     admin_users = models.ManyToManyField(settings.AUTH_USER_MODEL, blank=True, related_name='admin_organizations')
     regular_users = models.ManyToManyField(settings.AUTH_USER_MODEL, blank=True,
                                            related_name='organization_memberships')
-    created_by = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='created_organizations',
-                                   null=True, blank=True, editable=False)
-    last_modified_by = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='modified_organizations',
-                                         null=True, blank=True, editable=False)
-    replaced_by = models.OneToOneField('self', null=True, blank=True, related_name='replaced_organization',
-                                       help_text=_('The organization that replaces this organization'))
+    created_by = models.ForeignKey(
```

Comment:
With these foreign keys, we might actually want to use `SET_NULL` instead of `CASCADE`, just to be on the safe side. Imagine e.g. a single user is deleted, we might not want to delete all organizations they have ever edited :)

Reply:
Good point. I'll change `created_by` and `last_modified_by` to `SET_NULL`. `replaced_by` can be on `CASCADE`, I assume?

Reply:
Hm. Probably safest to have SET_NULL with replacements too. This way, organizations can be deleted without affecting other organizations by accident.